src/components/UI/Text/InputText.vue

```diff
@@ -25,7 +25,7 @@
       <!-- left icon -->
       <div
         v-if="isSideIconEnabled"
-        class="z-10 absolute font-xl text-blueGray-300 bg-transparent rounded text-base items-center text-xl w-5 inset-y-1/4 left-1.5"
+        class="absolute font-xl text-blueGray-300 bg-transparent rounded text-base items-center text-xl w-5 inset-y-1/4 left-1.5"
```

Comment:
z-index was not having any effect, and this was coming on top of the tool tip when z-10 was present